src/utils/utils.ts

```diff
@@ -82,3 +82,16 @@ export const getAriaAttrs = ({
 
   return ariaAttributes;
 };
+
+/** Convert a hex color value to an rgb or rgba value */
+export const hexToRGB = (hex: string, alpha: number) => {
```

Comment:
Am I imagining this or do we have a similar function somewhere else?

Reply:
Yes, this same function was added to the dark mode feature branch.  I needed it here to get the `rgba` value, so I brought it over.